java/src/test/java/vw/learner/VWFloatLearnerTest.java

```diff
@@ -31,7 +46,11 @@
     @BeforeClass
     public static void globalSetup() throws IOException {
         try {
-            System.load(new File(".").getCanonicalPath() + "/target/vw_jni.lib");
```

Comment:
Are all of these lines actually necessary?  Can't we revert back to the one line that was there before?